### PR TITLE
Add Findbugs plugin for slf4j loggers

### PR DESCRIPTION
**Pull request by griechel**
@punya @uschi2000 

Include patterns from http://kengotoda.github.io/findbugs-slf4j/ (except the static logger check)

### CODE REVIEW

File: gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineFindBugs.groovy [hunk 1]
```diff
@@ -31,6 +31,8 @@ class BaselineFindBugs extends AbstractBaselinePlugin {
     private static final String DEFAULT_FINDBUGS_VERSION = '3.0.1'
     private static final String DEFAULT_EFFORT = 'max'
     private static final String ANTIPATTERNS_ARTIFACT = 'com.palantir.antipatterns:antipatterns:1.0-beta.4'
+    private static final String SLF4J_FINDBUGS_PLUGIN = 'jp.skypencil.findbugs.slf4j:bug-pattern:1.2.4@jar'
```
Comment (uschi2000) [inline gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineFindBugs.groovy:35]: why do we need the "@jar" component? isn't that the default

**Comment by punya** [comment]
@uschi2000 see https://github.com/KengoTODA/findbugs-slf4j/issues/48

File: gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineFindBugs.groovy (hunk 1 above)
Comment (punya) [reply gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineFindBugs.groovy:35]: @uschi2000 see https://github.com/KengoTODA/findbugs-slf4j/issues/48.

**Comment by uschi2000** [comment]
ok. can you add a comment with a link to the GH issue and punya's response
there?

**Comment by griechel** [comment]
yep

**Comment by griechel** [comment]
accidentally closed. whoops

**Pull request merged**